be/models.py

```diff
@@ -48,34 +49,81 @@ class SearchParams(BaseModel):
     sort_order: Literal["desc", "asc"] = "asc"
 
 
-# MaveDB.
+# Datasource definition.
+
+class FieldDefinition:
+    def __init__(self, name: str, type: type, filterable: bool = False):
+        self.name = name
+        self.type = type
+        self.filterable = filterable
+
+
+class DataSource:
+    def __init__(
+        self,
+        name: str,
+        fields: List[FieldDefinition],
```

Comment:
as we are using Python 3.12, there is no need to import List from typing - just use list[...]

Reply:
please also check that we are using list instead of List in other places as well

Reply:
Changed `List` to built-in `list` in all three places: https://github.com/EMBL-EBI-ABC/PerturbationCatalogue/pull/101/commits/928fd75ef78cf1d4f7b97fed154e4f862e98285b

Reply:
(And also checked that other files have no usage of `List`)